packages/liquidator/index.js

```diff
@@ -196,9 +202,37 @@ async function run({
     const gasEstimator = new GasEstimator(logger);
     await gasEstimator.update();
 
+    let dsProxyManager;
+    if (proxyTransactionWrapperConfig?.isUsingDsProxyToLiquidate) {
+      dsProxyManager = new DSProxyManager({
+        logger,
+        web3,
+        gasEstimator,
+        account: accounts[0],
+        dsProxyFactoryAddress:
+          proxyTransactionWrapperConfig?.dsProxyFactoryAddress || getAddress("DSProxyFactory", networkId),
+        dsProxyFactoryAbi: getAbi("DSProxyFactory"),
+        dsProxyAbi: getAbi("DSProxy")
+      });
+
+      await dsProxyManager.initializeDSProxy();
```

Comment:
What does this function do? Might be worth adding a comment about which side effects trigger

Reply:
it will load in an existing DSProxy for the account EOA if one already exists or create a new one for the user. I added a comment